site/content/docs/include/publish.en.md

```diff
@@ -25,3 +25,26 @@ Holds configuration for a single SNS topic.
 
 <span class="parent-field">publish.topics.topic.</span><a id="topic-name" href="#topic-name" class="field">`name`</a> <span class="type">String</span>  
 Required. The name of the SNS topic. Must contain only upper and lowercase letters, numbers, hyphens, and underscores.
+
+<span class="parent-field">publish.topics.topic.</span><a id="fifo" href="#fifi" class="field">`fifo`</a> <span class="type">Boolean or Map</span>  
```

Comment:
```suggestion
<span class="parent-field">publish.topics.topic.</span><a id="publish-topics-topic-fifo" href="#publish-topics-topic-fifo" class="field">`fifo`</a> <span class="type">Boolean or Map</span>  
```
would you mind fixing this for the other places including https://github.com/aws/copilot-cli/pull/4024/files#diff-d0eb80518ed6d9a2e69c9f2a9cbe2b53a457323cc55296d23d14f8a59d969972R26 too?